time timestamp
string topic
string time_shipped
string message
string sim_time

